config.json

```diff
@@ -978,13 +995,13 @@
   "tags": [
     "execution_mode/compiled",
     "paradigm/procedural",
+    "typing/static",
     "platform/android",
     "platform/ios",
     "platform/linux",
     "platform/mac",
     "platform/windows",
     "runtime/language_specific",
-    "typing/static",
```

Comment:
Why was `typing/static` reordered here? It looks like it's otherwise in alphabetical order.

Reply:
On the v3 website (currently) the first three tags are listed beneath the track name in the listing of tracks.
This means that "platform/android" is one of the listed items. This changes that, making the tag used more relevant to the language.

I based this change in the thought that although C can indeed be used on Android, Android is not necessarily required to create C programs.

Reply:
:+1: